server/src/test/java/org/kaazing/gateway/server/config/parse/GatewayConfigParserTest.java

```diff
@@ -229,7 +232,7 @@ public void testValidWsInactivityTimeout() throws Exception {
         }
     }
 
-    @Test(expected = GatewayConfigParserException.class)
+    @Test(expected = GatewayConfigParserException.class) @Ignore
```

Comment:
I updated the Ignore annotation to give a reason why the test is no longer valid.  This is a candidate for removal altogether, but I didn't want to do that yet since maybe a better way of specifying the options that allow for XSD validation would be suggested.